### PR TITLE
Add Support for all Parity Chains

### PR DESCRIPTION
**Pull request by hayesgm**
This patch simply adds basic support for all chains that are supported for parity. As we aren't currently verifying proof of work nor authority, it's fairly simple to add new chains. In the future, to properly enable these chains, we will need to make sure we can handle the given engine (POW or POA). Also, even if we do not support a chain, this means we could write a nice message like "We do not support Ellacoin due to ...".

### CODE REVIEW

File: apps/blockchain/lib/blockchain/chain.ex [hunk 1]
```diff
@@ -463,9 +463,26 @@ defmodule Blockchain.Chain do
     iex> Blockchain.Chain.id_from_string("jungle")
     :not_found
   """
-  @spec id_from_string(String.t()) :: {:ok, atom()} | :not_found
+  @spec id_from_string(String.t()) ::
+          {:ok, :kovan | :tobalaba | :ropsten | :foundation | :expanse | :ellaism | :musicoin}
+          | :not_found
   def id_from_string(chain_name) do
```
Comment (InoMurko) [inline apps/blockchain/lib/blockchain/chain.ex]: hmmm, would this be more readable maybe?
```
  @doc """
   Given a string (e.g. user input), returns either a valid atom referencing a chain or `{:error, :not_found}`.	   
 	 
   ## Examples
 	 
     iex> Blockchain.Chain.id_from_string("ropsten")
     {:ok, :ropsten}	   
iex> Blockchain.Chain.id_from_string("jungle")
{:error, :not_found}
"""
@spec id_from_string(String.t()) ::
          {:ok, :kovan | :tobalaba | :ropsten | :foundation | :expanse | :ellaism | :musicoin}
          | {:error, :not_found}
def id_from_string("kovan"), do: {:ok, :kovan}
def id_from_string("tobalaba"), do: {:ok, :tobalaba}
def id_from_string("expanse"), do: {:ok, :expanse}
def id_from_string("ellaism"), do: {:ok, :ellaism}
def id_from_string("musicoin"), do: {:ok, :musicoin}
def id_from_string("ropsten"), do: {:ok, :ropsten}
def id_from_string("foundation"), do: {:ok, :foundation}
def id_from_string(_), do: {:error, :not_found}
```

File: chains/tobalaba.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		"0x0000000000000000000000000000000000000008": { "builtin": { "name": "alt_bn128_pairing", "activate_at": 7250000, "pricing": { "alt_bn128_pairing": { "base": 100000, "pair": 80000 } } } }
+	},
+
+	"nodes": [
+		"enode://eda34244538d72f42605a6fc8b8a34b15714c683989e8b29dc9e7a2b2088da490a5b32f2c149bec5a5c482bf03ec2c4f38b833ae31e36fcb26fb05fd094b2a88@18.197.33.9:30303",
+		"enode://12e903e900137b02b22e01f7918bd6e7310773c313e4e577281f35597e394a3e0b54c7314a8970a9776c5a3e5dc4daee289215dea3897bcb6d5cf0bb1dd2d356@18.197.31.231:30303",
+		"enode://423fdb91b37ec0714af0c19f625ec4af3ada2844367a36e45a05703577a84f7f0e9483585d4950a35c9e3738dba8c6abd7e1ce278d9a1f3f28065bc009f409cd@52.221.203.209:30303",
+		"enode://a9327d37d07799817d4a3e13d49fb4f5cc1486d4adf3ec8a6b98be62c4d7a5453914a5139dbe124809a388514cb0be37f9fa799539abe2250672f6d3d778b821@18.191.209.251:30303"
+	]
+}
```
Comment (InoMurko) [inline chains/tobalaba.json]: EOF

File: chains/expanse.json [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+			"balance": "1000000000000000000000000"
+		},
+		"c075fa11f85bda3aaba67106226aaf086ac16f4e": {
+			"balance": "100000000000000000000000"
+		},
+		"93decab0cd745598860f782ac1e8f046cb99e898": {
+			"balance": "10000000000000000000000"
+		}
+	}
+}
```
Comment (InoMurko) [inline chains/expanse.json]: EOF

**Comment by InoMurko** [comment]
Would it make sense to try and sync these chains somewhere?

File: chains/musicoin.json [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+				"pricing":{
+					"alt_bn128_pairing":{
+						"base":100000,
+						"pair":80000
+					}
+				}
+			}
+		}
+	}
+}
```
Comment (MattMSumner) [inline chains/musicoin.json]: Here's a blog post describing why GitHub warns about missing newlines at the end of files: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Comment by hayesgm** [comment]
@MattMSumner @InoMurko addressed your comments

**Pull request merged**